EnableRules is mentioned by name only

aaronjones opened the pull request:
I trust that this is the link to the appropriate file given the description in the gallery.  If not, please provide the correct link.

azurecla commented:
Hi __@SilviaDoomra__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sdoomra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@torteng 
fyi

torsteng commented:
This change is not (yet) correct. We have to wait until the corresponding change rolls out into production. Can we hold the change?

torsteng commented:
Hi Ross, Silvia,

This change is not yet live. Hence, the change is not correct. Can we revert the change and wait until this has become live in production?

Thanks,
Torsten

From: Ross McAllister [mailto:notifications@github.com]
Sent: Thursday, January 21, 2016 12:08 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Torsten Grabs <torsteng@microsoft.com>
Subject: Re: [azure-content] Update sql-database-elastic-transactions-overview.md (#5429)


@torsteng<https://github.com/torsteng>
fyi

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/5429#issuecomment-173693431>.

SilviaDoomra commented:
Hello everyone,

I have submitted a request to revert the change until the changes are in production. Torsten will submit the final doc changes once its ready.

Thanks,
Silvia Doomra | Program Manager II | Azure SQL Database | Microsoft Corporation

I support TEALS. How about you? Particpate in Giving, your way at //Give.

From: Torsten Grabs [mailto:notifications@github.com]
Sent: Thursday, January 21, 2016 1:23 PM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Silvia Doomra <sdoomra@microsoft.com>
Subject: Re: [azure-content] Update sql-database-elastic-transactions-overview.md (#5429)

Hi Ross, Silvia,

This change is not yet live. Hence, the change is not correct. Can we revert the change and wait until this has become live in production?

Thanks,
Torsten

From: Ross McAllister [mailto:notifications@github.com]
Sent: Thursday, January 21, 2016 12:08 PM
To: Azure/azure-content <azure-content@noreply.github.com<mailto:azure-content@noreply.github.com>>
Cc: Torsten Grabs <torsteng@microsoft.com<mailto:torsteng@microsoft.com>>
Subject: Re: [azure-content] Update sql-database-elastic-transactions-overview.md (#5429)


@torsteng<https://github.com/torsteng>
fyi

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/5429#issuecomment-173693431>.


—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/5429#issuecomment-173711582>.

aaronjones committed:
EnableRules is mentioned by name only

I trust that this is the link to the appropriate file given the description in the gallery.  If not, please provide the correct link.

abshamsft reviewed (changes requested):
@aaronjones Thanks for your contribution to the NPM article. Can you please use https://aka.ms/npmpowershellscript as the link to the EnableRules script?

aaronjones committed:
Update log-analytics-network-performance-monitor.md

abshamsft reviewed (approved):
Thanks for making the change, @aaronjones

aaronjones committed:
Merge branch 'master' into patch-2
